### PR TITLE
Breakout the config of channel(s) into confi ini

### PR DESCRIPTION
**Pull request by sacres**
Enabling channel(s) to be set in config.ini

### CODE REVIEW

**Comment by bbriggs** [comment]
LGTM, just need to merge the latest from master to resolve the conflict in Factoids

**Comment by sacres** [comment]
@bbriggs Done. Thanks! Errant trackpad closed this; reopened. X_X

**Comment by bastelfreak** [comment]
@sacres can you squash this please?

File: Local/Factoids.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def handle(self, message):
             txt = 'IRC is not Twitter – please do not use @name to address people. use name,' \
             ' or name: instead. Using @ also prevents you from tab-completing nicks in your IRC client'
         elif command == '!source':
+            txt = "You can find me on: 'https://github.com/voxpupuli/thevoxfox'"
```
Comment (bbriggs) [inline Local/Factoids.py:34]: Looks like some weirdness happening here. I see `txt ==` on 35 and `txt =` on 34.

**Review by bbriggs — CHANGES_REQUESTED** [review]

**Comment by sacres** [comment]
@bastelfreak You would think I could _but_ git log shows these 2 commits and git rebase not. Haven't come across that before. 
Closing, fix_#21 replaces this.

**Pull request closed without merging**